compiler/ir/backend.common/src/org/jetbrains/kotlin/ir/builders/declarations/declarationBuilders.kt

```diff
@@ -227,7 +226,7 @@ internal fun IrFactory.buildValueParameter(builder: IrValueParameterBuilder, par
         return createValueParameter(
             startOffset, endOffset, origin,
             IrValueParameterSymbolImpl(wrappedDescriptor),
-            name, index, type, varargElementType, isCrossInline, isNoinline, isHidden
+            name, index, type, varargElementType, isCrossInline, isNoinline, isHidden, false
```

Comment:
I would also suggest to support new flag in builder too.

Reply:
Yes, good idea. Done.